exposestrategy/ingress.go

```diff
@@ -139,6 +141,12 @@ func (s *IngressStrategy) Add(svc *api.Service) error {
 			UID:        svc.UID,
 		})
 	}
+
+	if s.ingressClass != "" {
+		ingress.Annotations["kubernetes.io/ingress.class"] = s.ingressClass
+		ingress.Annotations["nginx.ingress.kubernetes.io/ingress.class"] = s.ingressClass
+	}
```

Comment:
I guess in the future it might be nice to make the annotation keys configurable too, so we could support other ingress controllers apart from Nginx.  Just a comment really, we can address that in another PR